### PR TITLE
#142 notify the user when a new secret is added

### LINKED ISSUE #142: let's notify the user when a new secret is added
Let's do `human.notify()` from `secrets.add()`, in order to inform the user that a new secret has been just added to his account.

### PR DESCRIPTION
**Pull request by Suban05**
Closes #142

### CODE REVIEW

**Comment by Suban05** [comment]
@Yegorov please review

File: test/baza/test_secrets.rb [hunk 1]
```diff
@@ -50,4 +50,23 @@ def test_simple_scenario
...
     assert(secrets.each.to_a.empty?)
   end
+
+  def test_notify_user_after_creating
+    loog = Loog::Buffer.new
+    human = Baza::Humans.new(fake_pgsql, tbot: Baza::Tbot::Fake.new(loog)).ensure(fake_name)
+    n = fake_name
+    k = fake_name
+    v = fake_name * 10
+    id = human.secrets.add(n, k, v)
```
Comment (Yegorov) [inline test/baza/test_secrets.rb:60]: @Suban05 Looks as [redundant vars](https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html), may be better as:
```ruby
id = human.secrets.add(fake_name, 'secret_key', SecureRandom.uuid)
```

File: test/baza/test_secrets.rb [hunk 2]
```diff
@@ -50,4 +50,23 @@ def test_simple_scenario
...
+    assert_includes(loog.to_s, "Secret with ID #{id} has been successfully added.")
+  end
+
+  def test_does_not_notify_user_after_fail_creating
+    loog = Loog::Buffer.new
+    human = Baza::Humans.new(fake_pgsql, tbot: Baza::Tbot::Fake.new(loog)).ensure(fake_name)
+    assert_raises do
+      human.secrets.add(nil, nil, nil)
+    end
+    assert_empty(loog.to_s)
```
Comment (Yegorov) [inline test/baza/test_secrets.rb:70]: @Suban05 Maybe need more cases, when [secret not added](https://github.com/zerocracy/baza/blob/master/objects/baza/secrets.rb#L84-L90)
And add what exception should raises, e. g.: `assert_raises(Baza::Urror)`

**Review by Yegorov — CHANGES_REQUESTED** [review]
@Suban05 Looks good, but there are several comments

**Comment by Suban05** [comment]
@Yegorov updated

**Comment by Suban05** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@Suban05 thanks!

**Comment by 0crat** [comment]
@Yegorov Thanks for your code review! Here's how your +4 points were calculated: +4 base points, -24 for self-review, +16 for hits-of-code (capped), -10 for <6 comments, +1.25 for 5 comments, and +16.75 to meet the minimum 4-point reward. Your [running balance](https://zerocracy.github.io/judges-action/zerocracy-vitals.html) is now +416. Keep up the good work! 💻

**Comment by 0crat** [comment]
@Suban05 Hey there! 👋 Great job on your contribution! You've earned +4 points this time around. Here's the breakdown: +4 base points, -4 because we only got 5 comments (we're aiming for at least 6), and +4 to keep you motivated. Remember, more [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) and active code reviews can boost your points next time. [Keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the contributions coming! Your current balance is [+408](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Let's aim higher for your next submission! 💪

**Comment by Yegorov** [comment]
@yegor256 
> -24 for self-review

What does it mean "self-review"?

**Pull request merged**